lib/pbench/test/unit/server/query_apis/test_table_of_content.py

```diff
@@ -0,0 +1,206 @@
+from http import HTTPStatus
+
+import pytest
+
+from pbench.server.api.resources.query_apis.datasets.table_of_content import (
+    TableOfContent,
+)
+from pbench.server.database.models.datasets import Dataset
+from pbench.test.unit.server.query_apis.commons import Commons
+
+
+class TestTableOfContent(Commons):
```

Comment:
Well, consistency is good; I would have much preferred using the (more conventional) plural form in both cases.

Reply:
Dave, that was my fault:  I gave her the choice.  🤣

I, too, would prefer the plural form, but that would require pervasive changes, and I'm guessing that we want to completely replace "table of content(s)" with something like "files" or "directory contents".